packages/mdc-select/README.md

```diff
@@ -164,12 +164,12 @@ Mixin | Description
 --- | ---
 `mdc-select-ink-color($color)` | Customizes the color of the selected item displayed in the select.
 `mdc-select-container-fill-color($color)` | Customizes the background color of the select.
-`mdc-select-label-color($color)` | Customizes the label color of the select in the unfocused state.
+`mdc-select-label-color($color, $opacity: 1)` | Customizes the label color of the select in the unfocused state.
 `mdc-select-focused-label-color($color, $opacity: 0.87)` | Customizes the label color of the select when focused. Changing opacity for the label when floating is optional.
 `mdc-select-bottom-line-color($color)` | Customizes the color of the default bottom line of the select.
 `mdc-select-focused-bottom-line-color($color)` | Customizes the color of the bottom line of the select when focused.
 
-> NOTE: To customize label color please see the [label readme](./label/README.md).
+> NOTE: To customize label color please see the [floating label readme](./../mdc-floating-label/README.md).
```

Comment:
I'm a little confused here... we say to see a separate readme for customizing label color, but there are two mixins directly above that seem to cover this?

Reply:
Maybe it would be word better as:

`NOTE: To further customize label color please see...`

There is fill-color that we do not cover in the mixins above.